tensorboard/webapp/runs/views/runs_table/runs_data_table.scss

```diff
@@ -26,3 +26,45 @@ $_circle-size: 12px;
   width: $_circle-size - 2px; // size minus border
   vertical-align: middle;
 }
+
+$_arrow_size: 16px;
+.full-screen-toggle {
+  opacity: 0;
+  position: absolute;
+  height: 100%;
+  // Ensure the button is on the right side then add 2px for the drag target.
+  left: calc(100% + 2px);
+  top: 0;
+  z-index: 10;
```

Comment:
Nit: This seems arbitrary. Is anything else at z-index 1 or 2?

Reply:
This is indeed arbitrary, I changed it to 1